drivers/flash/flash_stm32_ospi.c

```diff
@@ -1845,6 +1845,49 @@ static int stm32_ospi_write_status_register(const struct device *dev, uint8_t re
 	return ospi_write_access(dev, &s_command, regs_p, size);
 }
 
+static int stm32_ospi_program_addr_4b(const struct device *dev, bool write_enable)
+{
+	uint8_t statReg;
+	struct flash_stm32_ospi_data *data = dev->data;
+	OSPI_HandleTypeDef *hospi = &data->hospi;
+	uint8_t nor_mode = OSPI_SPI_MODE;
+	uint8_t nor_rate = OSPI_STR_TRANSFER;
+	OSPI_RegularCmdTypeDef s_command = ospi_prepare_cmd(nor_mode, nor_rate);
+
+	if (write_enable) {
+		if (stm32_ospi_write_enable(data, nor_mode, nor_rate) < 0) {
+			LOG_ERR("program_addr_4b failed to write_enable");
```

Comment:
I would suggest removing logs here or replacing them with `LOG_DBG`.
The error is captured upon invocation, and related output is generated.

Reply:
changed